mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -311,7 +311,6 @@ settings.backup_configuration.binary_log_enabled are both set to true.`,
 									"allocated_ip_range": {
 										Type:         schema.TypeString,
 										Optional:     true,
-										ForceNew:     true,
```

Comment:
Is the update case for this field something we can add a test for, or is it too hard to orchestrate?

Reply:
We can create a test on updating this field with a new private network, as I mentioned in the PR description, but just not within the same VPC. I will add one on this.